include/boost/process/detail/windows/child_handle.hpp

```diff
@@ -82,14 +82,14 @@ struct child_handle
         ::boost::detail::winapi::BOOL_ value;
         if (!::boost::detail::winapi::IsProcessInJob(proc_info.hProcess, nullptr, &value))
             throw_last_error("IsProcessinJob Failed");
-        return value;
+        return value==1; // TODO: correct Windows API usage is: value==TRUE but TRUE isn't defined at winapi
```

Comment:
That's not entirely correct. Afaik the guarantee you have is that `FALSE == 0` and `TRUE != 0`, which is also in the [documentation](https://msdn.microsoft.com/de-de/library/windows/desktop/ms684127.aspx) of the function, where it states  "If the function succeeds, the return value is nonzero."

Hence it should be `value != 0`.